internal/pkg/daemon/bpfrecorder/bpf/recorder.bpf.c

```diff
@@ -93,6 +96,7 @@ typedef struct __attribute__((__packed__)) event_data {
 const volatile char filter_name[MAX_COMM_LEN] = {};
 const volatile u32 exclude_mntns = 0;
 
+static const char RUNC_DONE[] = "runc:[2:INIT]";
```

Comment:
This will lock us only to runc. Other alternatives like crun which I think is used by crio-o, it will not be covered. It is worth to mention this somewhere into the docs when this gets merged.

Reply:
@saschagrunert do we have tests to cover crun for seccomp? I think this won't affect the number of syscalls recorded in that case.

Reply:
Yes the e2e tests for seccomp should use runc and crun.